tycho/context.py

```diff
@@ -251,14 +250,6 @@ def get_spec (self, app_id):
                 else:
                     logger.error (f"-- app {app_id}. failed to parse spec.")
                 raise e
-        else:
```

Comment:
This was added a long time ago. Maybe the log description is wrong but we noticed a bug where i think previously launched app volumes are appended to this spec object and also gets created for a different user launched apps. So this logic kind of clears those volumes for every launch request. I am not entirely sure will need to re-check.